docs(review): review docs 17/2023 (#1581)
scaleway/docs-content: compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx

> 1. Open a terminal and type the following command, to configure the Scaleway CLI on your machine:

Reviewer: ```suggestion
1. Open a terminal and type the following command to configure the Scaleway CLI on your machine:
```

    `scw init`

>     If you have already configured the CLI, you will see a message like the following:

Reviewer: ```suggestion
    If you have already configured the CLI, you will see a message similar to the following:
```

    dcd8d994-b859-4e9e-aa91-2ceca41663fb  nl-ams-1  x86_64
    ff48b73a-097d-4685-b996-d3ebe50636ea  fr-par-1  x86_64
    ```

> 2. Type the following command in your terminal to create an Instance:

Reviewer: ```suggestion
2. Run the following command to create an Instance:
```

## Creating an Instance local volume

> 1. Type the following command in your terminal to create an Instance volume with a size of 50GB:

Reviewer: ```suggestion
1. Run the following command to create an Instance volume with a size of 50GB:
```

    State             available
    Zone              fr-par-1
    ```

> 2. Write down the ID of your Instance, as you will need it in the next steps.

Reviewer: ```suggestion
2. Write down the ID of your Instance. You will need it in the next steps.
```

## Creating and attaching a block volume to an Instance

<Message type="important">
  Block volumes are only available for `DEV1`, `GP1` and `RENDER` offers.
</Message>

> 1. Type the following command in your terminal to create a [block volume](/compute/instances/concepts/#block-volumes/): 

Reviewer: ```suggestion
1. Run the following command to create a [block volume](/compute/instances/concepts/#block-volumes/): 
```

    Project           12345678-1234-1234-4321-123456789123
    State             available
    ```

> 2. Write down your block volume's ID as you will need it in the next steps.

Reviewer: ```suggestion
2. Write down your block volume's ID. You will need it in the next steps.
```

> 3. Type the following command in your terminal to attach your Block volume to your Instance. Make sure you replace `n4m3-0f-y0Ur-s3rv3r-id` by your server ID and `n4m3-0f-y0Ur-volume-id` by your volume ID.

Reviewer: ```suggestion
3. Type the following command in your terminal to attach your block volume to your Instance. Make sure you replace `n4m3-0f-y0Ur-s3rv3r-id` by your server ID and `n4m3-0f-y0Ur-volume-id` by your volume ID.
```

    `scw instance server attach-volume server-id=n4m3-0f-y0Ur-s3rv3r-id volume-id=n4m3-0f-y0Ur-volume-id`

>     You will see an output like the following:

Reviewer: ```suggestion
    You will see an output similar to the following:
```

  This action will delete all your volumes and data located on your storage server. To avoid losing data, it is recommended to perform regular snapshots.
</Message>

> Ensure that your Instance is powered off, and type the following command in your terminal. Replace "your-server" with the ID of your server.

Reviewer: ```suggestion
Ensure your Instance is powered off and then run the following command, replacing "your-server" with the ID of your server.
```
